src/app/shared/services/apis/gb3/gb3-print.service.ts

```diff
@@ -225,7 +263,11 @@ export class Gb3PrintService extends Gb3ApiService {
                 type: 'WMS',
                 url: mapItem.url,
                 layers: mapItem.layers,
-                custom_params: mapItem.customParams,
+                custom_params: {
```

Comment:
This is a slight change of the existing logic. Previously, if there are no custom params, we sent `custom_params: undefined`
Now, we're sending `custom_params: {}`

Maybe this would be more like the original behavior:
```suggestion
                custom_params: mapItem.customParams ? { ... } : undefined,
```